tests/main/lxd-no-fuse/task.yaml

```diff
@@ -48,9 +48,5 @@ execute: |
     lxd.lxc file push --quiet "$GOHOME"/snapd_*.deb "my-ubuntu/$GOHOME/"
     lxd.lxc exec my-ubuntu -- apt install -y "$GOHOME"/snapd_*.deb
 
-    echo "And validate that we get a useful error"
-    if lxd.lxc exec my-ubuntu snap install test-snapd-sh 2> err.txt; then
-        echo "snap install should fail but it did not?"
-        exit 1
-    fi
-    MATCH 'The "fuse" filesystem is required' < err.txt
+    echo "And validate snaps can be installed because fuse is a snapd dependency"
```

Comment:
Shouldn't this message say "...because fuse comes with core"?

Reply:
sure, I'll update that

Reply:
thanks for the review